File: week2/sort.java
package week2;

import java.util.*;

class Main{
    public static void main(String[] args) {
    	
    	ArrayList<Integer> list = new ArrayList<Integer>();
    	Scanner scanner = new Scanner(System.in);
    	int cnt = scanner.nextInt();
    	
    	for(int i=0; i<cnt; i++) {
    		int num = scanner.nextInt();
    		list.add(num);
    	}
    	Collections.sort(list);
    	for(int i:list) {
    		System.out.println(i);
    	}
    	scanner.close();
    }
}

File: week2/sort.py
#백준 2750
sort = []
n = int(input())

for i in range (n):
    sort.append(int(input()))

for i in range(len(sort)):
    for j in range(len(sort)):
        if sort[i] < sort [j]:
            sort[i], sort[j] = sort[j], sort[i]

for num in sort:
    print(num)